Adyen/UI/Form/Items/Text/FormTextItemView.swift

```diff
@@ -295,7 +295,7 @@ open class FormTextItemView<ItemType: FormTextItem>: FormValueItemView<String, F
         }
     }
     
-    private func resetValidationStatus() {
+    open func resetValidationStatus() {
```

Comment:
I'm keeping the open scope since the rest of methods on `FormTextItemView` are also `open`. No problem changing it to `public` scope. What do you think?

Reply:
Change to `internal` scope.